v1/protocols/hyper.ts

```diff
@@ -54,12 +54,14 @@ export class HyperProtocol implements Protocol<Static<typeof HyperProtocolFields
     const mirror = fs.mirror(drive)
 
     // TODO: Should we log the changes somewhere like IPFS?
-    await mirror.done()
+    for await (const change of mirror) {
+      ctx?.logger.debug(`[hyper] ${change.op}: ${change.key}`)
+    }
 
     const raw = drive.url
     const link = raw
-    // TODO: Pass in the gateway URL in the config first
-    const gateway = 'oops'
+    // TODO: Pass in gateway from config
+    const gateway = `https://hyper.hypha.coop/hyper/${drive.id}/`
```

Comment:
Should we make this configurable if people don't want to use our gateway?

Reply:
Yeah, I was hoping to leave the gateway config as a TODO since it'd require a bunch of type changes. 😅

Reply:
I'll make an issue to track this unless you think it should be a blocker for the PR